wp-content/themes/amti/js/tinymce.js

```diff
@@ -27,14 +27,44 @@
                             minWidth: 700,
                             minHeight: 200
                         },
-                    
+
                     ],
                     onsubmit: function( e ) {
                         editor.insertContent( '[tocSidebar]'+e.data.tableOfContentsSidebar+'[/tocSidebar]\n[tocMain]'+e.data.tableOfContentsMain+'[/tocMain]');
                     }
                 });
             },
         });
+        editor.addButton( 'definition', {
+            text: 'Definition',
+            icon: false,
+            onclick: function() {
+                editor.windowManager.open( {
+                    title: 'Definition/Jargon:',
+                    width: 400,
+                    height: 100,
+                    body: [
+                        {
+                            type: 'textbox',
+                            multiline: false,
+                            name: 'term',
+                            label: 'Term',
+                            placeholder: 'This is the term'
+                        },
+                        {
+                            type: 'textbox',
+                            multiline: false,
+                            name: 'definition',
+                            label: 'Definition',
+                            placeholder: 'This is the definition'
+                        }
+                    ],
+                    onsubmit: function( e ) {
+                        editor.insertContent( "[define definition='"+e.data.definition+"']"+e.data.term+"[/define]");
```

Comment:
Can you please change this line so the definition attribute is wrapped in double quotes, not single ones? It should be `definition="definition goes here"`. This will help avoid conflicts if they want to use an apostrophe in the text.